src/app/cluster/duck/sagas.ts

```diff
@@ -250,11 +252,11 @@ function* startWatchingClusterAddEditStatus(action) {
   const statusToDispatch = addEditResult || createAddEditStatus(
     AddEditState.TimedOut, AddEditMode.Edit);
 
-  yield put(Creators.setClusterAddEditStatus(statusToDispatch));
+  yield put(ChangeActions.setClusterAddEditStatus(statusToDispatch));
 }
 
 function* watchClusterAddEditStatus() {
-  yield takeLatest(Creators.watchClusterAddEditStatus().type, startWatchingClusterAddEditStatus);
```

Comment:
+1, I think this is more canonical and a convention we should enforce going forwards.